source/_docs/configuration/remote.markdown

```diff
@@ -25,13 +25,13 @@ To set up a remote instance, you will need to set up Port Forwarding on your rou
 <div class='note'>
 General Instructions on Port Forwarding can be found by searching `<router model> port forwarding instructions`. 
 </div>
-<div class='note important'>
+<div class='note warning'>
 It is not advised to set up direct NAT without HTTPS enabled.  Always enable HTTPS.
 </div>
 
 One potential problem with port forwarding on a residential connection is most internet service providers use dynamic IP addresses. This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. You can solve this by using a Dynamic DNS service.
 <div class='note'>
-[DuckDNS](https://www.duckdns.org/) provides free Dynamic DNS. [Google Domains](https://domains.google), or [No IP](https://noip.com) provide both the Domain and the Dynamic DNS required for Home Assistant. 
+ [DuckDNS](https://www.duckdns.org/) provides free Dynamic DNS. [Google Domains](https://domains.google), or [No IP](https://noip.com) provide both the Domain and the Dynamic DNS required for Home Assistant. 
 </div>
```

Comment:
I don't think this should be in a note block. This can be done in regular text, as there is not need to alarm the user for these links.

Reply:
I thought class=warning was for alarm?  This is just a note.

Reply:
Sure, but avoiding them is in general a good idea. There is no need to emphasize these on this page, as part of a regular text it would still work as well.